main.cpp

```diff
@@ -444,7 +533,32 @@ Thing 9) Studio Server Hardware
     2) connect to internet
     3) create backup
  */
-
+struct serverHardware
+{
+  float connectionSpeed;
+  int numberOfEthernetIn;
+  float amountOfPowerCons;
+  float dailyDataTransfer;
+  std::string brand;
+
+  struct hardware
+  {
+    std::string brand = "DELL";
+    std::string model = "DL360";
+    int modelYear = 2014;
+    double ramType = 256.33;
+    std::string ssdType = "SSD";
+
+    void connectToPower(std::string conectionType = "EU Type");
+    void openTheHardware();
+    void enterTheOS(std::string nickname = "vsl", int osPassword = 283742);
+  };
+  void sendFiles(std::string fileType = ".wav", float fileSize = 462.f);
+  void connectTheWifi(std::string wifiName = "sikildim", int wifiPassword = 35359200);
+  void createBackUp(serverHardware hardware2);
+
+  hardware dataTransferDone;
+};
```

Comment:
Fix the following:
- Type Name capitalization
- Indentation relative to the `{ }`
- [ ] revise